testutil/pr/trackpr/main.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/obolnetwork/charon/testutil/pr"
+)
+
+func main() {
+	err := pr.Verify()
```

Comment:
why verify?

Reply:
Not sure if that's important but basically the `Track()` function assumes that the PR is "valid" and doesn't verify it's validity.

This is how i'hv structured it:
`Verify()` simply verifies the PR
`Track()` simply tracks the PR
    - `unticketed()` checks if the PR needs to be tracked.

Reply:
follow the linux mantra: "do only one thing", so remove `verify`